src/main/java/org/apache/commons/imaging/ImageFormats.java

```diff
@@ -43,24 +48,29 @@ public enum ImageFormats implements ImageFormat {
     XBM("xbm"),
     XPM("xpm");
 
-    private final String[] extensions;
+    private final List<String> extensions;
 
-    ImageFormats(final String ...extensions) {
-        this.extensions = extensions;
+    ImageFormats(final String... extensions) {
+        this.extensions = Arrays.stream(extensions).collect(Collectors.toList());
```

Comment:
Why not `java.util.Arrays.asList(...)`? `Collectors.toList()` makes not guarantees about the created list, see [documentation](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Collectors.html#toList()):
> There are no guarantees on the type, mutability, serializability, or thread-safety of the List returned

If the intention here is to create a copy of the array, you could also do `Arrays.asList(extensions.clone())`; that would make the intention clearer then.
(But I don't think there is a need to copy the array here.)